src/xcode/ENA/ENA/Source/Scenes/CreateAntigenTestProfile/Cells/CreateAntigenTestProfileInputCell.swift

```diff
@@ -37,6 +37,7 @@ class CreateAntigenTestProfileInputCell: UITableViewCell, ReuseIdentifierProvidi
 
 		birthdayPicker = UIDatePicker()
 		birthdayPicker.locale = Locale(identifier: "de_DE")
+		birthdayPicker.timeZone = TimeZone.current
```

Comment:
```suggestion
		birthdayPicker.timeZone = TimeZone.utcTimeZone
```

Would make the most sense for me since we are supposed to handle all dates in UTC.